LogonResponse.h

```diff
@@ -0,0 +1,20 @@
+#ifndef LOGIN_REQUEST_H
+#define LOGIN_REQUEST_H
+#include<string>
+#include "DataLinkMessage.h"
+class LogonResponse: public DataLinkMessage
```

Comment:
better to add space between the includes and the following code, 
```suggestion
#ifndef LOGIN_REQUEST_H
#define LOGIN_REQUEST_H

#include<string>

#include "DataLinkMessage.h"

class LogonResponse: public DataLinkMessage
```